pkg/api/list.go

```diff
@@ -39,6 +39,7 @@ var VersionList = []Version{
 	{"apps/v1beta2", "StatefulSet", "v1.9.0", "v1.16.0", "apps/v1"},
 
 	{"extensions/v1beta1", "NetworkPolicy", "v1.9.0", "v1.16.0", "networking.k8s.io/v1"},
+	{"extensions/v1beta1", "Ingress", "v1.9.0", "v1.16.0", "networking.k8s.io/v1beta1"},
```

Comment:
This was deprecated in `v1.14.0` and will be removed in `v1.22.0`. It is confusingly put in the same article as the 1.16 removals, but does specify those versions: https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/

Reply:
Thanks for the catch. Fixed.